### DISCUSSION TITLE
how to uninstall helix which was installed from source?i

### DISCUSSION THREAD

**Discussion by aspizu** [helix-editor/helix#5228] [Q&A] [answered]
installed using 

`cargo install --path helix-term    `

**Comment by the-mikedavis** [comment, marked as answer]
You can use [`cargo uninstall`](https://doc.rust-lang.org/cargo/commands/cargo-uninstall.html) from the `helix-term` directory within the repository to remove the `hx` binary.

Then remove the `~/.config/helix/runtime` (unix), `%AppData%\helix\runtime` (Windows cmd), or `$Env:AppData\helix\runtime` (Windows PowerShell) directory to remove the runtime directory.

If you want to remove all files related to Helix (any configuration files or custom themes), you can remove the entire `~/.config/helix` / `%AppData%\helix` / `$Env:AppData\helix` directory and you can remove the log file `helix.log` from the `~/.cache/helix` / `%AppData\Roaming\helix` / `$Env:AppData\helix` directory. But if you are switching from a from-source build to a build from a package manager, you will want to skip this step to preserve your configuration files.
